e2e/protocol.test.js

```diff
@@ -269,6 +269,22 @@ describe('executeScript', () => {
     })
 })
 
+it('can handle navigation and command execution', async () => {
+    await browser.navigateTo('http://guinea-pig.webdriver.io')
+    expect(typeof (await browser.getTitle())).toContain('string')
+
+    const link = await browser.findElement('link text', 'two')
+    await browser.elementClick(link[ELEMENT_KEY])
+    expect(typeof (await browser.getTitle())).toContain('string')
+
+    await browser.back()
+    expect(typeof (await browser.getTitle())).toContain('string')
+
+    const githubLink = await browser.findElement('link text', 'GitHub Repo')
+    await browser.elementClick(githubLink[ELEMENT_KEY])
```

Comment:
is it possible to verify that title is `GitHub`?

Reply:
see comment below